synapse/config/server.py

```diff
@@ -1014,6 +1022,24 @@ def generate_config_section(
         # act as if no error happened and return a fake session ID ('sid') to clients.
         #
         #request_token_inhibit_3pid_errors: true
+
+        # A list of domains that the domain portion of 'next_link' parameters
+        # must match.
+        #
+        # This parameter is optionally provided by clients while requesting
+        # validation of an email or phone number, and maps to a link that
+        # users will be automatically redirected to after validation
+        # succeeds. Clients can make use this parameter to aid the validation
+        # process.
+        #
+        # The whitelist is applied whether the homeserver or an
+        # account_threepid_delegate is handling validation.
```

Comment:
```suggestion
        # identity server is handling validation.
```

Reply:
Heh, I actually wrote this so that when we (hopefully) remove the `account_threepid_delegate` option, I remember to change this docstring as well.

You can *technically* handle verification without a full identity server, but just something that handles the verification endpoints. I've been careful to avoid writing "identity server" in other places because of this.

Reply:
According to its documentation, `account_threepid_delegate` only works if the address matches a trusted identity server. Also, one could argue that something that implements the verification endpoint is _technically_ an identity server, it just doesn't implement the IS spec fully, but can still be used to manage and verify identity to some extent.
Though more to the point, my concern is that "an account_threepid_delegate" doesn't really mean anything and is pretty confusing imho.

Reply:
Wow, yeah my bad, the docs for `account_threepid_delegates` mention identity servers all over the place :| Also that bit is a bit confusing, as we have another option for trusted identity servers, which isn't relevant here.

And yeah, good point. They may be very small processes, but they're still using the IS API.